docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -307,13 +307,25 @@ To set required TOKEN and SECRET values, consider using Gitea's built-in [genera
 
 ## SSH Container Passthrough
 
-Since SSH is running inside the container, SSH needs to be passed through from the host to the container if SSH support is desired. One option would be to run the container SSH on a non-standard port (or moving the host port to a non-standard port). Another option which might be more straightforward is to forward SSH connections from the host to the container.
+Since SSH is running inside the container, SSH needs to be passed through from the host to the container if SSH support is desired. One option would be to run the container SSH on a non-standard port (or moving the host port to a non-standard port). Another option which might be more straightforward is to forward SSH connections from the host to the container - that is, users of Gitea ssh to the Gitea user on the host and the host forwards those connections to the docker.
 
-There are multiple ways of doing this - however, all of these require some information about the docker being passed to the host.
+To understand what needs to happen, you first need to understand what happens without passthrough. So we will try to explain this:
+
+1. The client adds their SSH public key to Gitea using the webpage.
+2. Gitea will add an entry for this key to the `.ssh/authorized_keys` file of its running user, `git`.
+3. This entry has the public key, but also has a `command=` option. It is this command that Gitea uses to match this key to the client user and manages authentication.
+4. The client then makes an SSH request to the SSH server using the `git` user, e.g. `git clone git@domain:user/repo.git`.
+5. The client will attempt to authenticate with the server, passing one or more public keys in turn to the server.
```

Comment:
I'm really confused by the `in turn` part, I don't know what's the meaning of it within this sentence.

Reply:
This is, perhaps, too British, an idiom. (I'm surprised though it seems to be very standard English.)

<dl>
<dt>in turn</dt>
<dd>One after the other, in order, in succession</dd>
</dl>

The client SSH will present the user's keys to the server SSH one by one. If a key authenticates, the client SSH will not present another. 

The meaning of the `in turn` above is to give this impression.

Reply:
```suggestion
5. The client will attempt to authenticate with the server, passing one or more public keys in succession to the server.
```